src/nni_manager/common/utils.ts

```diff
@@ -276,7 +276,7 @@ function prepareUnitTest(): void {
     Container.snapshot(TrainingService);
     Container.snapshot(Manager);
 
-    setExperimentStartupInfo(true, 'unittest', 8080);
+    setExperimentStartupInfo('new', 'unittest', 8080);
```

Comment:
better to define macro for it, and use capital

Reply:
Fixed, use a constant to describe experiment startup info.